content/en/database_monitoring/setup_oracle/_index.md

```diff
@@ -12,17 +12,35 @@ is_beta: true
 {{< /site-region >}}
 
 <div class="alert alert-info">
-The features described on this page are in private beta. Contact your Customer Success Manager for information on the required Agent build and installation instructions.
+The features described on this page are in private beta.
 </div>
 
-### Oracle versions supported
+# Supported Oracle versions, features and architectures
 
-|  | Self-hosted | RDS |
-|--|-------------|-----| 
-| Oracle 19c | {{< X >}} | {{< X >}} |
-| Oracle 21c | {{< X >}} | {{< X >}} |
+- Versions: 19c and 21c
+- Deployment configurations: self-managed, RDS, RAC, Exadata
+- Architecture: multi-tenant
 
-The following deployment configurations are supported: self-managed, RDS, single-node, and multi-tenant. Note these do not include RAC and Exadata or legacy architectures.
+The following deployment configurations, components, and features are not supported: Oracle Autonomous Database, ASM, and Data Guard.
+
+The Oracle versions below 19c are not supported.
+
+# Supported Agent versions
+
+The following custom build is supported: `7.44.1~dbm~oracle~beta~0.28`
+
+# Prerequisites
+
+1. **Install the Oracle integration.** Install the Oracle integration in your organization by opening the [Oracle integration](https://app.datadoghq.com/integrations/oracle) in the Datadog App and clicking "Install." This will install an Oracle dashboard in your account which can be used to monitor the performance of your Oracle databases.
```

Comment:
```suggestion
1. **Install the Oracle integration**: In Datadog, install the [Oracle integration][1] for your organization. This will install an Oracle dashboard in your account which can be used to monitor the performance of your Oracle databases.
```